<launch>
      <node pkg="tf" type="static_transform_publisher" name="base_to_camera1_broadcaster" args="0 0.05 0.55 0 0 0 os_imu cam0 100" />
      <node pkg="tf" type="static_transform_publisher" name="base_to_camera2_broadcaster" args="0 -0.05 0.55 0 0 0 os_imu cam1 100" />
      <node pkg="tf" type="static_transform_publisher" name="base_to_laser_broadcaster" args="0 0 0.55 0 0 0 os_imu os_lidar 100" />
      <node name="airsimnode_static" pkg="airsim" type="airsimnode_static.py" respawn="false" output="screen">

            <param name="rate" type="int" value="30"/>
            <param name="imu_rate" type="int" value="90" />
            
            <param name="camera1_active" type="int" value="1" />
            <param name="camera2_active" type="int" value="1" />
            <param name="imu_active" type="int" value="1" />
            <param name="pose_active" type="int" value="1" />
            <param name="gpulidar_active" type="int" value="1" />

            <param name="lidar_frame_id" type="string" value="os_lidar" />
            <param name="camera1_frame_id" type="string" value="cam0" />
            <param name="camera2_frame_id" type="string" value="cam1" />
            <param name="imu_frame_id" type="string" value="os_imu" />
            <param name="pose_frame_id" type="string" value="world" />

            <param name="topic_scene_camera1" type="string" value="/airsim/camera/left/rgb/image" />
            <param name="topic_segmentation_camera1" type="string" value="/airsim/camera/left/segmentation/image" />
            <param name="topic_depth_camera1" type="string" value="/airsim/camera/left/depth/image" />
            <param name="topic_scene_camera2" type="string" value="/airsim/camera/right/rgb/image" />
            <param name="topic_segmentation_camera2" type="string" value="/airsim/camera/right/segmentation/image" />
            <param name="topic_depth_camera2" type="string" value="/airsim/camera/right/depth/image" />
            <param name="topic_pose" type="string" value="/airsim/gtpose" />
            <param name="topic_imu" type="string" value="airsim/imu" />
            <param name="topic_lidar" type="string" value="airsim/lidar" />

            <param name="vehicle_name" type="string" value="airsimvehicle" />
            <param name="camera1_name" type="string" value="leftcamera" />
            <param name="camera2_name" type="string" value="rightcamera" />
            <param name="lidar_name" type="string" value="lidar" />
            <param name="imu_name" type="string" value="imu" />

            <param name="camera1_sceneonly" type="int" value="0" />
            <param name="camera2_sceneonly" type="int" value="0" />
            <param name="camera1_mono" type="int" value="0" />
            <param name="camera2_mono" type="int" value="0" />
            <param name="camera_height" type="int" value="540" />
            <param name="camera_width" type="int" value="960" />
            <param name="camera_segmentation_disabled" type="int" value="0" />
            <param name="camera_encoding_quality" type="int" value="0" />
      </node>
</launch>
